src/remoted/manager.c

```diff
@@ -1101,7 +1109,6 @@ static void read_controlmsg(const char *agent_id, char *msg)
         file++;
 
         // If group was not got, guess it by matching sum
-
         mdebug2("Agent '%s' with group '%s' file '%s' MD5 '%s'",agent_id,group,file,md5);
         if (!f_sum) {
```

Comment:
I think this if is not necessary. At this point, `f_sum` will be always NULL.

Reply:
Done.